README.md

```diff
@@ -103,52 +115,63 @@ platforms (for example, `linux-64`), which can be used with [Conda] or [mamba].
 
 With Conda you could run:
 
-    $ conda install -c conda-forge python=3 cmake boost gmp mpfr \
+```
+conda install -c conda-forge python=3 cmake boost gmp mpfr \
          gettext libedit texinfo doxygen graphviz
-
+```
 ### Ubuntu
 
 If you're going to build on Ubuntu, `sudo apt-get install ...` the
 following packages (current as of Ubuntu 18.04):
 
-    $ sudo apt-get install build-essential cmake doxygen \
-         libboost-system-dev libboost-dev python3-dev gettext git \
-         libboost-date-time-dev libboost-filesystem-dev \
-         libboost-iostreams-dev libboost-python-dev libboost-regex-dev \
-         libboost-test-dev libedit-dev libgmp3-dev libmpfr-dev texinfo tzdata
+```
+sudo apt-get install build-essential cmake doxygen \
+    libboost-system-dev libboost-dev python3-dev gettext git \
+    libboost-date-time-dev libboost-filesystem-dev \
+    libboost-iostreams-dev libboost-python-dev libboost-regex-dev \
+    libboost-test-dev libedit-dev libgmp3-dev libmpfr-dev texinfo tzdata
+```
 
 ### Debian
 
 Debian 10 (bullseye), Debian 11 ("bullseye"), Debian testing and Debian
 unstable (sid) contain all components needed to build ledger.  You can
 install all required build dependencies using the following command:
 
-    $ sudo apt-get install build-essential cmake autopoint texinfo python3-dev \
-         zlib1g-dev libbz2-dev libgmp3-dev gettext libmpfr-dev \
-         libboost-date-time-dev libboost-filesystem-dev \
-         libboost-graph-dev libboost-iostreams-dev \
-         libboost-python-dev libboost-regex-dev libboost-test-dev
+```
+sudo apt-get install build-essential cmake autopoint texinfo python3-dev \
+    zlib1g-dev libbz2-dev libgmp3-dev gettext libmpfr-dev \
+    libboost-date-time-dev libboost-filesystem-dev \
+    libboost-graph-dev libboost-iostreams-dev \
+    libboost-python-dev libboost-regex-dev libboost-test-dev
+```
 
 ### Fedora
 
 You can install all required build dependencies under Fedora using the
 following command (tested with Fedora 32):
 
-    $ sudo dnf install boost-date-time boost-devel boost-filesystem \
-        boost-iostreams boost-python3-devel boost-regex boost-system \
-        boost-test cmake doxygen gettext git gmp-devel libedit-devel \
-        mpfr-devel python3-devel texinfo tzdata
-
+ ```
+sudo dnf install boost-date-time boost-devel boost-filesystem \
+    boost-iostreams boost-python3-devel boost-regex boost-system \
+    boost-test cmake doxygen gettext git gmp-devel libedit-devel \
+    mpfr-devel python3-devel texinfo tzdata
+```
 ## Building
 
 The next step is preparing your environment for building.  While you can use
 `cmake .` and make, I've prepared a script that does a lot more of the
 footwork for you:
 
-    $ ./acprep update
+```
+./acprep update
+```
     # or, if you want to use the Boost libraries with suffix -mt, install in
     # $HOME/local and build with 2 processes in parallel
```

Comment:
Please unindent these 2 lines and remove the leading `#`, so they appear as regular text.